tests/framework/framework.go

```diff
@@ -31,12 +31,16 @@ const (
 // run-time flags
 var (
 	kubectlPath  *string
-	ocPath	     *string
+	ocPath       *string
 	cdiInstallNs *string
 	kubeConfig   *string
 	master       *string
 )
 
+type FrameworkConfig struct {
```

Comment:
Rename this struct to `Config`. It's in the framework pkg so prepending _Framework_ is a bit redundant, to a degree. See my example of calling NewFramework below...